networkx/algorithms/minors.py

```diff
@@ -9,22 +9,66 @@
 from networkx.exception import NetworkXException
 from networkx.utils import arbitrary_element
 
-__all__ = ["contracted_edge", "contracted_nodes", "identified_nodes", "quotient_graph"]
+__all__ = [
+    "contracted_edge",
+    "contracted_nodes",
+    "equivalence_classes",
+    "identified_nodes",
+    "quotient_graph"
+]
 
 chaini = chain.from_iterable
 
 
 def equivalence_classes(iterable, relation):
-    """Returns the set of equivalence classes of the given `iterable` under
-    the specified equivalence relation.
+    """Returns equivalence classes of `relation` when applied to the elements
+    of `relation`.
```

Comment:
I'm trying to get this to one line (I know... quotient graph isn't one line either....)
But also, `relation` shouldn't appear twice.

```
"""Returns equivalence classes of `relation` applied to elements of `iterable`

```

Reply:
Sorry, this was a typo - it's in one line now.